demo/WebexMeetingWidgetDemo.jsx

```diff
@@ -58,12 +63,21 @@ export default function WebexMeetingWidgetDemo({token}) {
           <SelectOption value="dark" label="Dark" />
           <SelectOption value="light" label="Light" />
         </Select>
+        <div className="webex-theme-selector">Layout</div>
```

Comment:
rename this to something more general like webex-dropdown or webex-select or webex-select-control